lib/std/net.zig

```diff
@@ -331,6 +339,10 @@ pub const Address = extern union {
                 try std.fmt.format(context, Errors, output, "]:{}", port);
             },
             os.AF_UNIX => {
```

Comment:
Do all platforms have an `os.AF_UNIX` definition?

Reply:
Windows has `AF_UNIX` but WASI doesn't, which is weird? should I make a `const AF_UNIX = if (@hasDecl(os, "AF_UNIX")) os.AF_UNIX else -1;` or something? sounds hacky at best.

Reply:
Windows should have `sockaddr_un` too, from https://devblogs.microsoft.com/commandline/af_unix-comes-to-windows/
> The ‘sockaddr_un’ structure is used for defining the address of a unix socket. In the Windows implementation of the unix socket, we have kept the name, definition and semantics of the unix socket address the same as that in Linux, to make cross-platform development easier.

Reply:
Then that leaves WASI left. Should this PR take care of adding `sockaddr_un` to windows' bits?

Reply:
> Then that leaves WASI left. 

Let WASI bit-rot. It's a half-done spec that doesn't even support networking yet. Existing WASI code will continue to work just fine due to Zig's lazy analysis, since no existing WASI code calls the std.net APIs.

> Should this PR take care of adding `sockaddr_un` to windows' bits?

Sure, if you want to. Not required for merging though.